ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -478,6 +510,36 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- always_run: false
+  as: rosa-hcp-network-optional-presubmit
```

Comment:
Could you help me understand the meaning of `optional` here? 

> optional Set to true to make the job not block merges.

If you do not want this job to be a merge block in Prow CI, you should set `optional: true`.